feat(content): Show Arachi Cartoonist Your Leviathan (#11118)
endless-sky/endless-sky: data/coalition/coalition missions.txt

				`	"In that case, I'll head to their home shipyard and see what I can bring back for you."`
					goto end
				`	"Alright, it's a bit of a long trip to <planet>, but if you're sure, let's head to my ship and I'll show you your bunk."`
				`	"I happen to own a ship matching that description, a Betelgeuse Leviathan. Would you like to see it?"`
					goto hasleviathan
					to display
						has "ship model: Leviathan"
			`	"Oh, leave I cannot. Whether far away or not this <planet> you speak of is, allowed to leave Coalition space, we are not," he says. "Go on without me you must, Captain."`
			label end
			`	He thanks you and wishes you safe travels on your journey to <planet>.`
				accept
			label hasleviathan
			action
				"coalition jobs" += 2
				payment 75940
			`	Though reading emotions from a giant spider proves challenging, Drugub appears to be star-struck.`
			`	"Own a ship, a 'Leviathan,' you do? To your hangar, we must go!"`

> 			`	After a brief walk to the hangar where your ship is parked, Drugub immediately begins crawling over the Leviathan. It seems what you've provided to him is far more substantial than any secondhand reference could possibly provide. He spends a while examining the ship up close before hopping down and taking many pictures from every conceivable angle -- as well as drawing a few quick sketches.`

Reviewer: ```suggestion
			`	After a brief walk to the hangar where your ship is parked, Drugub immediately begins crawling over the Leviathan. It seems what you've provided him with is far more substantial than any secondhand reference could possibly convey. He spends a while examining the ship up close before hopping down and taking many pictures from every conceivable angle -- as well as drawing a few quick sketches.`
```

				`	"In that case, I'll head to their home shipyard and see what I can bring back for you."`
					goto end
				`	"Alright, it's a bit of a long trip to <planet>, but if you're sure, let's head to my ship and I'll show you your bunk."`

> 					goto itsalongtrip
> 				`	"I happen to own a ship matching that description, a Betelgeuse Leviathan. Would you like to see it?"`
> 					goto hasleviathan
> 					to display
> 						has "ship model: Leviathan"
> 			label itsalongtrip

Reviewer: ```suggestion
				`	"I happen to own a ship matching that description, a Betelgeuse Leviathan. Would you like to see it?"`
					goto hasleviathan
					to display
						has "ship model: Leviathan"
```
`goto` and `label` aren't necessary if there isn't a different label in between them. It can improve readability sometimes though, so it's your choice on this.

Author: If I were to write my own missions, I'd prefer to have each choice in a complex tree like this labeled. But as this isn't my mission, I'll roll with your suggestion here. I should still test it again with the new commits to make sure it doesn't mess up.

			`	"Oh, leave I cannot. Whether far away or not this <planet> you speak of is, allowed to leave Coalition space, we are not," he says. "Go on without me you must, Captain."`
			label end
			`	He thanks you and wishes you safe travels on your journey to <planet>.`
				accept
			label hasleviathan
			action
				"coalition jobs" += 2
				payment 75940

> 			`Though reading emotions from a giant spider proves challenging, Drugub appears to be star-struck.`

Reviewer: ```suggestion
			`	Though reading emotions from a giant spider proves challenging, Drugub appears to be star-struck.`
```

> 			`	"Own a ship, a 'Leviathan,' you do? To your hangar, we must go."`

Reviewer 2: ```suggestion
			`	"Own a ship, a 'Leviathan,' you do? To your hangar, we must go!"`
```

> 			`	After a brief walk to the hangar where your ship is parked, Drugub immediately begins crawling over the Leviathan. It seems what you've provided to him is far more substantial than any secondhand reference could possibly provide. He spends a while examining the ship up close before hopping down, proceeding to take pictures from every concievable angle.`

Reviewer 2: ```suggestion
			`	After a brief walk to the hangar where your ship is parked, Drugub immediately begins crawling over the Leviathan. It seems what you've provided to him is far more substantial than any secondhand reference could possibly provide. He spends a while examining the ship up close before hopping down, proceeding to take many pictures, as well as draw a few quick sketches, from every conceivable angle.`
```

> 			`	Once he's finished, he hands you your payment of <payment> and spends some time with you still, asking you all the questions he can think of about the ship.`

Reviewer: ```suggestion
			`	Once he's finished, he hands you your payment of <payment> and spends more time with you, asking you all the questions he can think of about the ship.`
```
This middle portion of the sentence feels a little off. I made an attempt to improve it.

Author: I think we could get away with keeping the word 'some'
> Once he's finished, he hands you your payment of <payment> and spends some more time with you, asking you all the questions he can think of about the ship.

Reviewer: Updated

> 			`	Once he's finished, he hands you your payment of <payment> and spends some more time with you, asking you all the questions he can think of about the ship.`

Reviewer: Given that it hasn't mentioned how much you'll be paid, I believe "your" isn't the correct word here.
```suggestion
			`	Once he's finished, he hands you a payment of <payment> and spends some more time with you, asking you all the questions he can think of about the ship.`
```

			`	"Very beautiful, this 'Leviathan' is. Thinking of many ways to draw it, I already am," he says. "Maybe a flagship of one of my characters, make it I will..."`
			`	He spends some more minutes listening to you after he asks that you describe how your ship performs, thanks you again, and prepares to leave.`
			`	"Oh, before I go, Captain <last>, use the name of your own ship for that 'Leviathan,' may I? Just as a small thank you."`
			choice
				`	"Of course! It's called <ship>."`
				`	"The name is nothing special. I wouldn't even get the reference of it if I read your comic in the future, honestly."`
					goto noship
			`	"<ship> it is! Do it justice I will, Captain. Thanks again."`
				decline
			label noship

> 			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`

Reviewer: ```suggestion
			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again, Captain."`
```

				accept
			label noship
			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`
				accept
			label leviathan
			`Drugub is excitedly waiting for you when you return. Though reading emotions from a giant spider proves challenging, Drugub appears to be star-struck when you mention you brought back a ship matching what he described.`

> 			`	"Own a ship, a 'Leviathan,' you do? To your hangar, we must go."`

Reviewer 2: ```suggestion
			`	"Own a ship, a 'Leviathan,' you do? To your hangar, we must go!"`
```

> 			`	After a brief walk to the hangar where your ship is parked, Drugub immediately begins crawling over the Leviathan. It seems what you've provided to him is far more substantial than any secondhand reference could possibly provide. He spends a while examining the ship up close before hopping down, proceeding to take pictures from every concievable angle.`

Reviewer 2: ```suggestion
			`	After a brief walk to the hangar where your ship is parked, Drugub immediately begins crawling over the Leviathan. It seems what you've provided to him is far more substantial than any secondhand reference could possibly provide. He spends a while examining the ship up close before hopping down, proceeding to take many pictures, as well as draw a few quick sketches, from every conceivable angle.`
```

> 			`	After a brief walk to the hangar where your ship is parked, Drugub immediately begins crawling over the Leviathan. It seems what you've provided to him is far more substantial than any secondhand reference could possibly provide. He spends a while examining the ship up close before hopping down and taking many pictures from every conceivable angle -- as well as drawing a few quick sketches.`

Reviewer: ```suggestion
			`	After a brief walk to the hangar where your ship is parked, Drugub immediately begins crawling over the Leviathan. It seems what you've brought to him is far more substantial than any secondhand reference could possibly provide. He spends a while examining the ship up close before hopping down and taking many pictures from every conceivable angle -- as well as drawing a few quick sketches.`
```

Author: I think I'd prefer keeping provided for the first instance, and changing the second instance to 'achieve' or 'convey.'

> 			`	Once he's finished, he hands you your payment of <payment> and spends some more time with you, asking you all the questions he can think of about the ship.`

Reviewer: "your" is correct here because you've already done work to receive it.
(feel free to resolve this once you've read it)

> 			`	Once he's finished, he hands you your payment of <payment> and spends some time with you still, asking you all the questions he can think of about the ship.`

Reviewer: ```suggestion
			`	Once he's finished, he hands you your payment of <payment> and spends more time with you, asking you all the questions he can think of about the ship.`
```
